tests/main/interfaces-network-control/task.yaml

```diff
@@ -121,6 +99,12 @@ execute: |
     echo "And a command can be executed in the context of the namespace"
     network-control-consumer.cmd ip netns exec test-ns ip link list | MATCH "veth1"
 
+    echo "Then the snap command can modify the network configuration"
+    network-control-consumer.cmd ip neigh add "$ARP_ENTRY_ADDR" lladdr aa:aa:aa:aa:aa:aa dev veth0
```

Comment:
is there a reason for using `veth0` instead of checking the default interface as before `INTERFACE=$(get_default_iface)`?

Reply:
we add veth0 for the test and we remove it in restore, so any changes to ARP tables done for that interface will be gone after the test completes